Source/EasyNetQ/IServiceProvider.cs

```diff
@@ -49,5 +49,7 @@ public interface IServiceRegister
         IServiceRegister Register<TService, TImplementation>()
             where TService : class
             where TImplementation : class, TService;
+
+        IServiceRegister UseDelayedMessageExchange();
```

Comment:
Even if I understand the fluent interface you're adding, I don't know if we would like to change this base service interface @mikehadlow what do you think?